compiler/damlc/tests/daml-test-files/ImplementsNonInterface.daml

```diff
@@ -0,0 +1,17 @@
+-- Copyright (c) 2020, Digital Asset (Switzerland) GmbH and/or its affiliates.
+-- All rights reserved.
+
+-- Test the serializability checker for Daml-LF.
+-- We use the template typeclass and instances directly as otherwise the error
+-- is caught prior due to missing Eq and Show instances.
+
+-- @ERROR range=17:5-17:20; cannot implement 'Bool' because it is not an interface
```

Comment:
My bad, I forgot to mention that you also need to add another magic comment so these files only get tested against versions that support interfaces (currently only the `1.dev` version)

```suggestion
-- @SINCE-LF-FEATURE DAML_INTERFACE
-- @ERROR range=17:5-17:20; cannot implement 'Bool' because it is not an interface
```